ton_client/client/src/contracts/run.rs

```diff
@@ -305,13 +305,22 @@ pub(crate) fn local_run(context: &mut ClientContext, params: ParamsOfLocalRun) -
 
     let key_pair = params.key_pair.map(|pair| pair.decode()).transpose()?;
 
-    let account = params.account.as_ref().map(|acc| 
+    let account = params.account
```

Comment:
DO NOT use formatter in functional PRs. If you want to format code acording to some scheme you should make separate PR for that

Reply:
ok